tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -340,6 +350,13 @@ TangoRosNode::TangoRosNode() : run_threads_(false) {
   } catch (const image_transport::Exception& e) {
     LOG(ERROR) << "Error while creating image transport publishers" << e.what();
   }
+  // Initialize start_of_service_T_area_description as identity.
+  tf::StampedTransform start_of_service_T_area_description =
+      tf::StampedTransform(tf::Transform(tf::Quaternion(0, 0, 0, 1)),
+                           ros::Time::now(),
```

Comment:
Why not publish this frame when it first becomes available? Then this discussion becomes obsolete.